dec.py

```
@@ -0,0 +1,95 @@
+        print(text[:1000])
+    except Exception:
+        print(decompressed[:1000])
+    print("-" * 60)
+
+    # Save to file if not already saved
+    try:
+        with open("decompressed_output_zlib.txt", "wb") as f:
+            f.write(decompressed)
+        print("\n✓ Full decompressed data saved to 'decompressed_output_zlib.txt'")
+    except Exception:
+        pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)